functions/exercise2/index.html

```diff
@@ -5,14 +5,19 @@
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <title>excercise2</title>
+    <link rel="stylesheet" href="style.css" />
     <script src="script.js"></script>
 </head>
 
 <body>
     <h1>Palindrome</h1>
-    Phrase:<input id="phrase" />
+    <input id="phrase" />
+
     <button onclick="showResult()">Click here to check if your phrase is palindrome</button>
-    <output id="result" />
+
+    <div class="resultStyle">
```

Comment:
You did not refer to my comment from previous review:
"Do we have to use both div and output here? Can't we use only one of them?"